Open Judge System/Data/OJS.Data/OjsDbContext.cs

```diff
@@ -26,6 +26,7 @@ public OjsDbContext()
         protected OjsDbContext(string nameOrConnectionString)
             : base(nameOrConnectionString, throwIfV1Schema: false)
         {
+            this.DbContext.Database.CommandTimeout = 240;
```

Comment:
Create an app settings env variable for this

Reply:
Extracted a Setting variable, so that Web will use CommandTimeout value from there, while OJS.LocalWorker will work as before. As we talked, the change was a bit more specific, because of the EfMigrations requirements for an empty public constructor, as well as SimpleInjector's requirements of only 1 Public constructor per class.